native-code/ios/index.md

```diff
@@ -165,7 +165,7 @@ To build WebRTC for use in a native iOS app, it's easiest to build
 with the actual location of your generated build files.
 
 ~~~~~ bash
-ninja -C out/ios rtc_sdk_framework_objc
+ninja -C out/ios framework_objc
```

Comment:
I really think we should have suffixes and since this is not yet pushed, can't you just land another CL in open source renaming the few targets that still have prefix instead of suffix? Then update this?